ppr-ui/src/composables/mhrRegistration/useHomeOwners.ts

```diff
@@ -76,6 +82,12 @@ export function useHomeOwners (isMhrTransfer: boolean = false) {
   const getHomeTenancyType = (): HomeTenancyTypes => {
     // Groups
     const groups = getTransferOrRegistrationHomeOwnerGroups().filter(owner => owner.action !== ActionTypes.REMOVED)
+    // Variable to track if owners has a valid combination of Executor/Trustee/Admin (ETA) Owners
+    let hasETAError = false
+    groups.forEach(group => {
+      if (hasETAGroupError(group)) hasETAError = true
```

Comment:
Maybe once we get the ETA error we don't have to continue iterating thru groups?

Reply:
refactored to use arr.some() instead